app/src/main/java/org/hackillinois/android/notifications/FirebaseTokenManager.kt

```diff
@@ -27,10 +29,12 @@ object FirebaseTokenManager {
     fun sendTokenToServerIfNew(context: Context) {
         if (isTokenNew(context)) {
             val token = readToken(context)
-            thread {
-                App.getAPI().sendUserToken(DeviceToken(token)).execute()
-                writeToken(context, DEFAULT_FIREBASE_TOKEN)
-                setIsTokenNew(context, false)
+            GlobalScope.launch(Dispatchers.IO) {
+                try {
+                    App.getAPI().sendUserToken(DeviceToken(token))
+                    writeToken(context, DEFAULT_FIREBASE_TOKEN)
+                    setIsTokenNew(context, false)
+                } catch (e: Exception) {}
```

Comment:
Can you add something like a log statement to the catch block just so it's cleaner/easier to trace?